src/main/java/org/primefaces/convert/CombinedDateTimePatternTokenVisitor.java

```diff
@@ -175,13 +175,12 @@ private static void visitDayOfWeekToken(int repetitions, JQueryDateTimePatternBu
      * More than three 'D' are interpreted as 'DDD'.
      */
     private static void visitDayOfYearToken(int repetitions, JQueryDateTimePatternBuilder builder) {
-        switch (repetitions) {
-            case 1:
-                builder.appendDayOfYearNoLeadingZeros();
-                break;
-            default: // 2+
-                builder.appendDayOfYearWithLeadingZeros();
-                break;
+        if (repetitions == 1) {
+            builder.appendDayOfYearNoLeadingZeros();
+        }
+        else {
+            // 2+
+            builder.appendDayOfYearWithLeadingZeros();
         }
```

Comment:
Sonar recommended removing the switch for if else for readaibility

Reply:
I used the switch-pattern in other methods too so always using that keeps it consistent, but yeah, this is fine as well.